File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TreeViews_View.cpp

#include "TreeViews_View.h"
#include "TreeViews_View.h"
#include <vector>
#include <memory>
#include "TreeViews_ViewMyTreeViewElementsRow.h"
#include "TreeViews_ViewMyStringRowHandlesRow.h"
#include "TreeViews_ViewMyParentIndexRowHandlesRow.h"
#include "TreeViews_ViewMyParentStringRowHandlesRow.h"
#include <string>

namespace widgetassertions
{
  int TreeViews_View::getMyTreeViewElementsTreeSelectedRow( ) 
  {
    return this->MyTreeViewElementsTreeSelectedRow;
  }
  bool TreeViews_View::getIsMyTreeViewElementsTreeVisible( ) 
  {
    return this->isMyTreeViewElementsTreeVisible;
  }
  bool TreeViews_View::getIsMyTreeViewElementsTreeEnabled( ) 
  {
    return this->isMyTreeViewElementsTreeEnabled;
  }
  std::vector<std::shared_ptr<TreeViews_ViewMyTreeViewElementsRow>> TreeViews_View::getMyTreeViewElementsTreeRows( ) 
  {
    return this->MyTreeViewElementsTreeRows;
  }
  bool TreeViews_View::getIsMyTreeViewElementsTreeDescriptionColumnVisible( ) 
  {
    return this->isMyTreeViewElementsTreeDescriptionColumnVisible;
  }
  std::vector<std::shared_ptr<TreeViews_ViewMyStringRowHandlesRow>> TreeViews_View::getMyStringRowHandlesTreeRows( ) 
  {
    return this->MyStringRowHandlesTreeRows;
  }
  std::string TreeViews_View::getMyStringRowHandlesTreeSelectedRow( ) 
  {
    return this->MyStringRowHandlesTreeSelectedRow;
  }
  std::vector<std::shared_ptr<TreeViews_ViewMyParentIndexRowHandlesRow>> TreeViews_View::getMyParentIndexRowHandlesTreeRows( ) 
  {
    return this->MyParentIndexRowHandlesTreeRows;
  }
  std::vector<std::shared_ptr<TreeViews_ViewMyParentStringRowHandlesRow>> TreeViews_View::getMyParentStringRowHandlesTreeRows( ) 
  {
    return this->MyParentStringRowHandlesTreeRows;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TreeViews_View.cs
/// <filename>
///     TreeViews_View.cs
/// </filename>
using System.Collections.Generic;
using widgetassertions;

namespace widgetassertions
{
    internal class TreeViews_View
  {
        private int MyTreeViewElementsTreeSelectedRow;
     
        public     virtual int getMyTreeViewElementsTreeSelectedRow()
    {
      return this.MyTreeViewElementsTreeSelectedRow;
    }
     
        private bool isMyTreeViewElementsTreeVisible;
     
        public     virtual bool getIsMyTreeViewElementsTreeVisible()
    {
      return this.isMyTreeViewElementsTreeVisible;
    }
     
        private bool isMyTreeViewElementsTreeEnabled;
     
        public     virtual bool getIsMyTreeViewElementsTreeEnabled()
    {
      return this.isMyTreeViewElementsTreeEnabled;
    }
     
        private System.Collections.Generic.List<widgetassertions.TreeViews_ViewMyTreeViewElementsRow> MyTreeViewElementsTreeRows = new System.Collections.Generic.List<widgetassertions.TreeViews_ViewMyTreeViewElementsRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.TreeViews_ViewMyTreeViewElementsRow> getMyTreeViewElementsTreeRows()
    {
      return this.MyTreeViewElementsTreeRows;
    }
     
        private bool isMyTreeViewElementsTreeDescriptionColumnVisible;
     
        public     virtual bool getIsMyTreeViewElementsTreeDescriptionColumnVisible()
    {
      return this.isMyTreeViewElementsTreeDescriptionColumnVisible;
    }
     
        private System.Collections.Generic.List<widgetassertions.TreeViews_ViewMyStringRowHandlesRow> MyStringRowHandlesTreeRows = new System.Collections.Generic.List<widgetassertions.TreeViews_ViewMyStringRowHandlesRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.TreeViews_ViewMyStringRowHandlesRow> getMyStringRowHandlesTreeRows()
    {
      return this.MyStringRowHandlesTreeRows;
    }
     
        private string MyStringRowHandlesTreeSelectedRow;
     
        public     virtual string getMyStringRowHandlesTreeSelectedRow()
    {
      return this.MyStringRowHandlesTreeSelectedRow;
    }
     
        private System.Collections.Generic.List<widgetassertions.TreeViews_ViewMyParentIndexRowHandlesRow> MyParentIndexRowHandlesTreeRows = new System.Collections.Generic.List<widgetassertions.TreeViews_ViewMyParentIndexRowHandlesRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.TreeViews_ViewMyParentIndexRowHandlesRow> getMyParentIndexRowHandlesTreeRows()
    {
      return this.MyParentIndexRowHandlesTreeRows;
    }
     
        private System.Collections.Generic.List<widgetassertions.TreeViews_ViewMyParentStringRowHandlesRow> MyParentStringRowHandlesTreeRows = new System.Collections.Generic.List<widgetassertions.TreeViews_ViewMyParentStringRowHandlesRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.TreeViews_ViewMyParentStringRowHandlesRow> getMyParentStringRowHandlesTreeRows()
    {
      return this.MyParentStringRowHandlesTreeRows;
    }
  }
}

